swift/Samples/SampleApp/Sources/DemoScreen.swift

```diff
@@ -0,0 +1,105 @@
+import Workflow
+import WorkflowUI
+
+
+struct DemoScreen: Screen {
+    let title: String
+    let color: UIColor
+    let onTitleTap: () -> Void
+
+    let refreshText: String
+    let isRefreshEnabled: Bool
+    let onRefreshTap: () -> Void
+}
+
+
+final class DemoViewController: ScreenViewController<DemoScreen> {
```

Comment:
same comment as the other: make `fileprivate` and move the `ViewRegistry` extension up